zboxcore/sdk/writemarker_mutex.go

```diff
@@ -64,6 +64,28 @@ func CreateWriteMarkerMutex(client *client.Client, allocationObj *Allocation) (*
 	}, nil
 }
 
+//the minimum of M blobbers must accept the marker
+func (m *WriteMarkerMutex) getMinimumAccept(T int) int {
```

Comment:
Let's not use the capital T as param variable, leave it for go generic using.

Reply:
fixed